hexa/workspaces/views.py

```diff
@@ -35,6 +36,18 @@ def credentials(request: HttpRequest, workspace_slug: str) -> HttpResponse:
 
     # TODO: Database / Filesystem credentials
 
+    # Database credentials
+    db_credentials = get_db_server_credentials()
+    env.update(
```

Comment:
I'd only add the `WORKSPACE_DB_URL` at first. @pvanliefland  WDYT ?

Reply:
The individual variables are needed for R - the typical postgres lib that our team uses cannot use a connection string url and need the individual parameters